### ISSUE TITLE
YouTube upload feed is dying

### ISSUE THREAD

**Issue by MichaelApproved** [KeithAndTheGirl/iOS#107]
Looks like YouTube is killing the RSS feed that we depend on for the upload stream. Is there another way for you to get a list of the most recently uploaded videos?

This is the feed. Notice the first video is "devicesupport"
http://gdata.youtube.com/feeds/api/users/keithandthegirl/uploads?&v=2&max-results=50

**Comment by Nick-R** [comment]
Yes, now latest video is "devicesupport"... Do you mean it appears later?

**Comment by MichaelApproved** [comment]
No, I'm saying the RSS feed that we're using is no longer supported by Google. They're putting in "devicesupport" in all old RSS feeds as way to explain that we're using an old RSS feed and that we need to update it.

I did some research and found that the YouTube feed should be changed to https://www.youtube.com/feeds/videos.xml?user=keithandthegirl

Here's more information http://www.reddit.com/r/youtube/comments/339idg/youtube_just_dropped_its_support_for_rssatom_feeds/

**Comment by Nick-R** [comment]
Oh, yes. Sorry, i didn't get this first time... I changed the code to aggregate new feed format. Please check in build 52.
